pkg/condition/condition.go

```diff
@@ -104,6 +104,21 @@ func GetMetav1ConditionStatus(status bool) metav1.ConditionStatus {
 	return metav1.ConditionFalse
 }
 
+func GetCondition(status *v2alpha1.DatadogAgentStatus, t string) *metav1.Condition {
+	idConditionComplete := getIndexForConditionType(status, t)
+	if idConditionComplete >= 0 {
+		return &status.Conditions[idConditionComplete]
+	}
+	return nil
+}
+
+func GetConditionDDAI(status *v1alpha1.DatadogAgentInternalStatus, t string) *metav1.Condition {
```

Comment:
This function is not used: is a test supposed to use it or it's added for parity + see above comment ?